src/msgraph_core/models/page_result.py

```diff
@@ -46,8 +46,17 @@ def get_field_deserializers(self) -> Dict[str, Callable[[ParseNode], None]]:
             object where each entry is a property key with its deserialization callback.
         """
         return {
-            "@odata.nextLink": lambda x: setattr(self, "odata_next_link", x.get_str_value()),
-            "value": lambda x: setattr(self, "value", x.get_collection_of_object_values(Parsable))
+            "@odata.nextLink":
+            lambda x: setattr(self, "odata_next_link", x.get_str_value()),
+            "value":
+            lambda x: setattr(
+                self,
+                "value",
+                x.get_collection_of_object_values(
+                    Parsable  # type: ignore
+                    # Bug. Should get a collection of primitive dictionary objects
```

Comment:
QQ: why would this be a collection of primitive dictionaries? Isn't it correct that its a collection of Parsables?

Reply:
Yes, nice catch. It's an error on my side